src/inputs/Table/components/TableToolbar/TableToolbar.js

```diff
@@ -50,6 +72,18 @@ TableToolbar.propTypes = {
    * Function used to export a file in Table. If undefined, export button is not displayed
    */
   onExport: PropTypes.func,
+  /*
+   * Function used to add a row in the Table. If it's not defined, he not not display the add row button
+   */
+  onAddRow: PropTypes.func,
+  /*
+   * Function used to remove selected rows in the Table. If it's not defined, he not not display the delete row button
+   */
+  onDeleteRow: PropTypes.func,
+  /*
+   * Boolean used to know if we had AddRow features enable in the webapp
```

Comment:
better phrasing: Boolean that defines whether AddRow feature is enabled or not

Reply:
Does not exist anymore